README.md

```diff
@@ -7,33 +7,36 @@
 
 
 ### **Features**
-* **Fully functional backend now linked to the remote AWS database**
-* **Developed authentication functionality**
-* **Developed the "Register" capability**
-* **Added category titles to the Resource page so articles could be grouped appropriately**
-* **Transitioned away from the side bubble navigation system and implemented a bottom navigation bar**
+* Fully functional backend now linked to the remote AWS database
+* Developed authentication functionality
+* Developed the "Register" capability
+* Added category titles to the Resource page so articles could be grouped appropriately
+* Transitioned away from the side bubble navigation system and implemented a bottom navigation bar
+
+### **Bug Fixes**
```

Comment:
Could also mention educational resources now have categories, even though it's in features it was a bug from last release so might be worth including it here too.